src/chrome/content/rules/Redmine.org.xml

```diff
@@ -17,23 +10,20 @@
 		- www.redmine.org
 
 -->
```

Comment:
IMO this comment and the next one is unnecessary, since there is no reason to list nonexistent domains and domains which set insecure cookies.